lib/config_cat/data_governance.ex

```diff
@@ -0,0 +1,11 @@
+defmodule ConfigCat.DataGovernance do
```

Comment:
Hmm, it's not that easy to define an enum.. :D

Reply:
Typespecs makes it a bit bloated 😄